static-analysis/teammates-stylelint.yml

```diff
@@ -530,3 +530,6 @@ rules:
 
     # Disallow animation names that do not correspond to a @keyframes declaration
     # no-unknown-animations
+
+    # One selector per line.
+    selector-list-comma-newline-after: always
```

Comment:
You should not actually need this. Somewhere in this YML file there should already be this rule, but commented out -- you only need to un-comment it.